payloads/library/ftp_exfiltrator/payload.txt

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Title:         FTP Exfiltrator
+# Author:        Nutt
+# Version:       1.0
+# Target:        Windows
+# 
+#Exfiltrates files from the users Documents folder
+#FTP's all files/folders to a specified FTP site named by the victim hostname.
+#Powershell FTP script will stay running after BashBunny is unplugged, once light turns green unplug and check FTP site.
+
+#Executes 1.ps1
+
+#Purple.........Setup
+#Red............Failed - Need to work on

```

Comment:
@dfanara 
That is a great idea! I was wanting to add a failed option because I know its possible for powershell to fail to run wheather blocked by UAC or even by group policy or something else.

I'll give it a shot and see

Thanks!